text.py

```diff
@@ -0,0 +1,34 @@
+from thirdai import bolt, dataset
```

Comment:
This file should be in the docker container, not here

Reply:
You can put it in the docker container and add the directory to the pythonpath, or something similar

Reply:
I want to keep it this way for now for simplicity purposes, especially since this may change after the data pipeline gets merged